build.js

```diff
@@ -107,7 +107,36 @@ for (const property of binaryProperties) {
 		`@unicode/unicode-${ UNICODE_VERSION }/Binary_Property/${ property }/code-points.js`
 	);
 	const set = regenerate(codePoints);
-	const output = `${ set.toCode() }\nmodule.exports = set;\n`;
+	const output = `${ set.toCode() }\nexports.characters = set;\n`;
+	fs.writeFileSync(fileName, output);
+}
+
+const propertiesOfStrings = [
+	'Basic_Emoji',
+	'Emoji_Keycap_Sequence',
+	'RGI_Emoji_Modifier_Sequence',
+	'RGI_Emoji_Flag_Sequence',
+	'RGI_Emoji_Tag_Sequence',
+	'RGI_Emoji_ZWJ_Sequence',
+	'RGI_Emoji',
+];
+for (const property of propertiesOfStrings) {
+	binaryProperties.push(property);
```

Comment:
Intuitively, I wouldn't list properties of strings under binary properties, but rather list them separately. `Binary_Property` -> `Property_of_Strings` (or the prior nomenclature `Sequence_Property`) or some such. WDYT?

Reply:
Done!